iam_policy/templates/jenkins_ecs_task_integration.json.tpl

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Is this policy specific for integration? If so then should remove the non-integration params as this policy. If not then maybe the name should be changed to make it more generic.

Reply:
I'm not sure tbh until I finish up creating prod Jenkins. I've guessed that this needs to be different but if the task policies end up the same then I'll probably merge them but for now, I'm just trying to keep everything as it is.